docs/en/latest/upgrade-guide.md

```diff
@@ -64,7 +64,7 @@ etcdserver:
 
 ##### Controller-Only Configuration Source
 
-In 2.0.0, all data plane configurations must originate from the Ingress Controller. Configurations via Admin API or any external methods are no longer supported and will be ignored or may cause errors.
+In APISIX Ingress Controller 2.0.0, a full-sync mode is used. All Data Plane configurations must originate from the Ingress Controller. Any additional route configurations made via the Admin API or other external methods will be ignored.
```

Comment:
> All Data Plane configurations must originate from the Ingress Controller. Any additional route configurations made via the Admin API or other external methods will be ignored.

I'm still confused by this. Ingress Controller 2.0 supports both Standalone and Admin API mode, so why `Any additional route configurations made via the Admin API or other external methods will be ignored.`?

Reply:
This is ambiguous and should be edited.

> Any additional route configurations made via the Admin API or other external methods will be ignored.

Manual Admin API operations outside of the ingress controller will be overridden by regular full updates. We used to allow those user-created resources to co-exist with those created by the ingress controller, which was frankly a mistake in itself.

The description should be made clearer to clarify that both Admin API and standalone mode are supported.

Reply:
> Manual Admin API operations outside of the ingress controller will be overridden by regular full updates. We used to allow those user-created resources to co-exist with those created by the ingress controller, which was frankly a mistake in itself.

That's right, I changed the description, please check again.